pkg/keyspace/tso_keyspace_group.go

```diff
@@ -137,6 +140,11 @@ func (m *GroupManager) Bootstrap() error {
 		m.groups[userKind].Put(group)
 	}
 
+	// It will only alloc node when the group manager is on API leader.
+	if m.client != nil {
+		m.wg.Add(1)
+		go m.allocNodesToAllKeyspaceGroups()
```

Comment:
If we transfer the leader, will it exit?

Reply:
Thanks, added ctx